app/src/main/res/values/strings.xml

```diff
@@ -22,4 +22,5 @@
     <string name="clearButton">Clear button</string>
     <string name="cancel">Cancel</string>
     <string name="sdk_sample_icon_description">SDK sample icon</string>
+    <string name="home_screen_title">Sample Categories</string>
```

Comment:
```suggestion
    <string name="home_screen_title">Sample categories</string>
```

Reply:
I feel like this change goes against the other top app bar titles which are capitalized (eg: `Augmented Reality`,`Utility Networks`

Keeping it as is, unless you would strongly recommend :)